lib/article_json/export/apple_news/elements/list.rb

```diff
@@ -0,0 +1,71 @@
+module ArticleJSON
+  module Export
+    module AppleNews
+      module Elements
+        class List < Base
+          include ArticleJSON::Export::Common::HTML::Elements::Base
+          include ArticleJSON::Export::Common::HTML::Elements::Text
```

Comment:
Are both of these needed to have access to `text_exporter`?

Reply:
I think they are actually possibly both redundant. Tests pass without them. I will do a manual test to confirm.

Reply:
Manual test passed. I will remove them now.

Reply:
Amended, thanks.